tests/testthat/helper-TealAppDriver.R

```diff
@@ -0,0 +1,26 @@
+# Import non-exported TealAppDriver from `teal` package
+TealAppDriver <- getFromNamespace("TealAppDriver", "teal") # nolint: object_name.
+
+# Helper function
+simple_teal_data <- function() {
+  data <- within(teal.data::teal_data(), {
+    require(nestcolor)
+    iris <- iris
+    mtcars <- mtcars
+  })
+  teal.data::datanames(data) <- c("iris", "mtcars")
+  data
+}
+
+simple_cdisc_data <- function(datasets = c("ADSL", "ADRS", "ADTTE")) {
+  datasets <- match.arg(datasets, several.ok = TRUE)
+  data <- within(teal.data::teal_data(), {
+    require(nestcolor)
+    ADSL <- teal.modules.general::rADSL
+    ADRS <- teal.modules.general::rADRS
+    ADTTE <- teal.modules.general::rADTTE
+  })
+
+  teal.data::datanames(data) <- datasets
+  data
```

Comment:
```suggestion
  datasets <- match.arg(datasets, several.ok = TRUE)
  within(teal.data::cdisc_data(join_keys = teal.data::default_cdisc_join_keys[datasets]), {
    require(nestcolor)
    ADSL <- teal.modules.general::rADSL
    ADRS <- teal.modules.general::rADRS
    ADTTE <- teal.modules.general::rADTTE
  })
```

Reply:
Good catch with the join_keys :+1:

Given that `cdisc_data` does nothing different than `teal_data`  if no named arguments are given, what do you think about keep using `teal_data()`?

```suggestion
  datasets <- match.arg(datasets, several.ok = TRUE)
  data <- within(
    teal.data::teal_data(join_keys = teal.data::default_cdisc_join_keys[datasets]), 
    {
      require(nestcolor)
      ADSL <- teal.modules.general::rADSL
      ADRS <- teal.modules.general::rADRS
      ADTTE <- teal.modules.general::rADTTE
    }
  )
```

Reply:
then you have to assign datanames .  I am ok with either of them.